### PR TITLE
Validate phone and email for Organization

### PR DESCRIPTION
**Pull request by snittoor**
Added semantic tags for organization from schema.org.
Validated email and phone number using regular expression while adding/updating organization.

Regular Expression Explanation for phone no:

    // can start with +/# ( 2 digits with or without ()) , can have -/./slash/or space between nos
    //eg. +(XX)-XXXX-XXXX , +XX XXXX XXXXX , #X/XXXX/XXXX/XXXX  , XX.XXXX.XXXX ,     XXXXXXXXXXXXX
    regEx: /^\+|#?(|[(][0-9]{2})|[)]\)?[-|.|/|\s ]?([0-9]{4})[-|.|/|\s ]?([0-9]{4})$/, 

1) \+|# : may or maynot have +or # in beginning.
2) ?(|[(][0-9]{2})|[)]\) : may or maynot have () , no for 0-9 , two digits
3) ?[-|.|/|\s ]?([0-9]{4})[-|.|/|\s ]?([0-9]{4})$ : the no can have - or . or / or space between then and 4 digits each. 
4) can enter just digits.
Issue #2034

### CODE REVIEW

File: organizations/client/form/form.html [hunk 1]
```diff
@@ -22,23 +22,25 @@ <h2 class="modal-title" id="organizationFormModalLabel">
...
           doc=organization
         }}
+        <div itemscope itemtype="http://schema.org/Organization">
           <div class="modal-body">
             <fieldset>
-              {{> afQuickField name='name' }}
-              {{> afQuickField name='url' }}
-              {{> afQuickField name='description' }}
+              <span itemprop="name"> {{> afQuickField name='name' }} </span>
+                <span itemprop="url"> {{> afQuickField name='url' }}</span>
```
Comment (brylie) [inline organizations/client/form/form.html]: Use consistent indentation.

File: organizations/client/form/form.html [hunk 2]
```diff
@@ -22,23 +22,25 @@ <h2 class="modal-title" id="organizationFormModalLabel">
           type=formType
           doc=organization
         }}
+        <div itemscope itemtype="http://schema.org/Organization">
           <div class="modal-body">
             <fieldset>
-              {{> afQuickField name='name' }}
-              {{> afQuickField name='url' }}
-              {{> afQuickField name='description' }}
+              <span itemprop="name"> {{> afQuickField name='name' }} </span>
```
Comment (brylie) [inline organizations/client/form/form.html]: To improve readability, nest HTML tags like so:

```html
<span itemprop="name">
  {{> afQuickField name='name' }}
</span>
```

**Comment by brylie** [comment]
Requested small improvements for readability.

File: organizations/collection/schema.js [hunk 3]
```diff
@@ -13,6 +13,9 @@ Organizations.schema = new SimpleSchema({
   },
   'contact.phone': {
     type: String,
+    // can start with +/# ( 2 digits with or without ()) , can have -/./slash/or space between nos
+    //eg. +(XX)-XXXX-XXXX , +XX XXXX XXXXX , #X/XXXX/XXXX/XXXX  , XX.XXXX.XXXX , XXXXXXXXXXXXX
+    regEx: /^\+|#?(|[(][0-9]{2})|[)]\)?[-|.|/|\s ]?([0-9]{4})[-|.|/|\s ]?([0-9]{4})$/,
```
Comment (brylie) [inline organizations/collection/schema.js]: I am wondering if this RegEx is too complicated.

**Comment by snittoor** [comment]
@brylie : I made the RegEx changes as discussed with Taija. These include having + ,( ) , space , + , . ,/ while entering the phone. Right now # and * are not included. Please have a review and let me know the changes.

File: organizations/collection/schema.js [hunk 4]
```diff
@@ -13,6 +13,9 @@ Organizations.schema = new SimpleSchema({
   },
   'contact.phone': {
     type: String,
+  // can start with +/# ( 2 digits with or without ()) , can have -/./slash/or space between
+  // nos eg. +(XX)-XXXX-XXXX , +XX XXXX XXXXX ,  XX.XXXX.XXXX ,     XXXXXXXXXXXXX
+    regEx: /^\+?(|[(][0-9]{2})|[)]\)?[-|.|/|\s ]?([0-9]{4})[-|.|/|\s ]?([0-9]{4})$/,
```
Comment (bajiat) [inline organizations/collection/schema.js]: Hi Sri,

Just wanted to make sure you understood my earlier comment. I would like you to control only which characters are allowed in the field, but not their order or grouping. But this is really hard for me to read, so I'm not sure what you are doing.

The reason why we do not want to control e.g. grouping is that the grouping of phone numbers depends on the locale (in this case country, not so much language). So just keep it simple.

Comment (snittoor) [inline organizations/collection/schema.js]: @bajiat : Hi Taija : Ok , will rework on the expression again.

File: organizations/collection/schema.js [hunk 5]
```diff
@@ -13,9 +13,7 @@ Organizations.schema = new SimpleSchema({
   },
   'contact.phone': {
     type: String,
-  // can start with +/# ( 2 digits with or without ()) , can have -/./slash/or space between
-  // nos eg. +(XX)-XXXX-XXXX , +XX XXXX XXXXX ,  XX.XXXX.XXXX ,     XXXXXXXXXXXXX
-    regEx: /^\+?(|[(][0-9]{2})|[)]\)?[-|.|/|\s ]?([0-9]{4})[-|.|/|\s ]?([0-9]{4})$/,
+    regEx: /^[0-9-+()/\s]+$/,
```
Comment (brylie) [inline organizations/collection/schema.js]: Remember to add a comment describing *why* this regular expression is written this way. What characters are you intending to allow?

File: organizations/collection/schema.js [hunk 6]
```diff
@@ -13,6 +13,9 @@ Organizations.schema = new SimpleSchema({
   },
   'contact.phone': {
     type: String,
+    // all numbers (0-9) , + , - , space ,/ are allowed and can appear anywhere in the phone numbers
+    //e.g. 754-3010,(541) 754-3010,1-541-754-3010,1-541-754-3010,191 541 754 3010,(089) / 636-48018
+    regEx: /^[0-9-+()/\s]+$/,
```
Comment (brylie) [inline organizations/collection/schema.js]: Remember to allow dot character '.'

**Pull request merged**